Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6.3
```

Comment:
Is this the right version? Not sure if we should use 2.4.1 or this one. If we're using this one I'll have to update my CI/CD config files

Reply:
I didn't have any problems to use the latest version, I think it is safe to do this move.